R/valley.R

```diff
@@ -61,11 +61,16 @@ get_dem <- function(bb, dem_source = "STAC", stac_endpoint = NULL,
                     stac_collection = NULL, crs = NULL,
                     force_download = FALSE) {
   # Check input
+  bbox <- as_bbox(bb)
+  checkmate::assert_choice(dem_source, "STAC")
+  checkmate::assert_character(stac_endpoint, null.ok = TRUE, len = 1)
+  checkmate::assert_character(stac_collection, null.ok = TRUE, len = 1)
+  checkmate::assert_multi_class(crs, c("numeric", "character"), null.ok = TRUE)
+  if (!is.null(crs)) checkmate::assert_vector(crs, len = 1)
```

Comment:
Should this simply become:
```suggestion
  crs <- as_crs(crs)
```